aspnetcore/migration/webapi.md

```diff
@@ -4,47 +4,164 @@ author: ardalis
 description: Learn how to migrate a web API implementation from ASP.NET 4.x Web API to ASP.NET Core MVC.
 ms.author: scaddie
 ms.custom: mvc
-ms.date: 12/05/2019
+ms.date: 05/20/2020
 no-loc: [Blazor, "Identity", "Let's Encrypt", Razor, SignalR]
 uid: migration/webapi
 ---
 # Migrate from ASP.NET Web API to ASP.NET Core
 
 By [Scott Addie](https://twitter.com/scott_addie) and [Steve Smith](https://ardalis.com/)
 
-An ASP.NET 4.x Web API is an HTTP service that reaches a broad range of clients, including browsers and mobile devices. ASP.NET Core unifies ASP.NET 4.x's MVC and Web API app models into a simpler programming model known as ASP.NET Core MVC. This article demonstrates the steps required to migrate from ASP.NET 4.x Web API to ASP.NET Core MVC.
+An ASP.NET 4.x Web API is an HTTP service that reaches a broad range of clients, including browsers and mobile devices. ASP.NET Core unifies ASP.NET 4.x's MVC and Web API app models into a unified programming model known as ASP.NET Core MVC. This article demonstrates the steps required to migrate from ASP.NET 4.x Web API to ASP.NET Core MVC.
 
 [View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/migration/webapi/sample) ([how to download](xref:index#how-to-download-a-sample))
 
+::: moniker range=">= aspnetcore-3.0"
```

Comment:
The current version ***ALWAYS*** goes first:

* In page links are broken in the 2nd and following moniker sections.
* It's the first thing customers (and us to) see when editing the file.

Reply:
@Rick-Anderson, thanks. The latest version is first in this case.  The content breakdown of versioning is 3.0 or higher on the top and version 2.2 or lower located at the bottom. The content at the top in this case is for  3.0 and 3.1.   There wasn't a 3.1 specific instruction or information, beyond the 3.1 download link provided in the include.  Thanks for pointing out the bad links, embarrassed I missed that.